esmvaltool/cmorizers/data/formatters/datasets/esacci_lst.py

```diff
@@ -1,160 +1,138 @@
-    # maybe the arbitrary difference should go on day cubes to
-    # take the timestamp to 12Z?
-    # not really an issue when using monthly files
-
-    result = iris.cube.CubeList([day_cube, night_cube]).concatenate_cube()
-
-    # This corrects the lonitude coord name issue
-    # This should be fixed in the next version of the CCI data
-    logger.info("Longitude coordinate correction being applied")
-    result.coords()[2].var_name = 'longitude'
-    result.coords()[2].standard_name = 'longitude'
-    result.coords()[2].long_name = 'longitude'
-
-    monthly_cube = result.collapsed('time', iris.analysis.MEAN)
-
-    # fix time coordinate bounds
-    monthly_co_time = monthly_cube.coord('time')
-
-    time_point = (datetime.datetime(year, month, 1, 0, 0) -
-                  datetime.datetime(1981, 1, 1, 0, 0, 0)).total_seconds()
-    monthly_co_time.points = time_point
-
-    num_days = monthrange(year, month)[1]
-    monthly_co_time.bounds = [
-        time_point, time_point + ((num_days - 1) * 24 * 3600)
-    ]
-    # should this be num_days or num_days-1 ### question for Valeriu or Axel
-    # or 23:59:59 ???
-
-    monthly_cube.attributes = {
-        'information': 'Mean of Day and Night Aqua MODIS monthly LST'
-    }
-
-    return monthly_cube
+                try:
+                    cubes = load_cubes(in_dir,
+                                       vals['file'],
+                                       year,
+                                       month,
+                                       vals['raw'],
+                                       )
+                except:
+                    logger.info(f'Problem: Month %s in %s' % (month, year))
+                    continue
+
+                #  make time coords
+                time_units = 'hours since 1970-01-01 00:00:00'
+                time_point = unit.date2num(datetime.datetime(year, month, 1),
+                                           time_units,
+                                           unit.CALENDAR_STANDARD
+                                           )
+
+                time_coord = iris.coords.DimCoord(time_point,
+                                                  standard_name='time',
+                                                  long_name='time',
+                                                  var_name='time',
+                                                  units=time_units,
+                                                  bounds=None,
+                                                  attributes=None,
+                                                  coord_system=None,
+                                                  circular=False
+                                                  )
+
+                cubes.attributes = {}
+                cubes.attributes['var'] = var
+
+                try:
+                    cubes.remove_coord('time')
+                except:
+                    logger.info('Coord fix issue %s' % cubes.long_name)
```

Comment:
here too - I'd be much happier if you caught the right exception eg coord not found etc; to the very least please be more specific in your logger message - it contains very little info about what went wrong. If you are unsure of the type and handling of exception, it's always better to leave it blow up, and not try catch it, mask it, then output an non-descriptive message